integration-cli/docker_cli_network_unix_test.go

```diff
@@ -783,6 +841,12 @@ func (s *DockerSuite) TestInspectApiMultipleNetworks(c *check.C) {
 	bridge := inspect121.NetworkSettings.Networks["bridge"]
 	c.Assert(bridge.IPAddress, checker.Equals, versionedIP)
 	c.Assert(bridge.IPAddress, checker.Equals, inspect121.NetworkSettings.IPAddress)
+
+	// cleanup
```

Comment:
@cpuguy83  you are right, I have already undone the cleanup, have a look again, thanks.